sdk/test/NetStandard/UnitTests/AWSSDK.UnitTests.Custom.NetStandard.csproj

```diff
@@ -6,7 +6,7 @@ This project file should not be used as part of a release pipeline.
 -->
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;netcoreapp3.1;net8.0</TargetFrameworks>
+        <TargetFrameworks>netcoreapp3.1;net8.0</TargetFrameworks>
```

Comment:
Netstandard is not a valid target for unit tests

Reply:
Ran the build system again and got the following error:

```
C:\Program Files\dotnet\sdk\8.0.300\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(266,5): error NETSDK1005: Assets file 'C:\codebuild\tmp\output\src3842370628\src\aws-sdk-net\sdk\test\NetStandard\UnitTests\obj\AWSSDK.UnitTests.Custom.NetStandard\project.assets.json' doesn't have a target for 'netstandard2.0'. Ensure that restore has run and that you have included 'netstandard2.0' in the TargetFrameworks for your project.
```

I believe it is likely because of this change which needs to be reverted. Although you are correct that it really isn't a correct target to run for unit tests it is part of the solution. So when the solution is built with:

```
dotnet build -c Release --no-incremental $buildTarget -f netstandard2.0 /p:AWSKeyFile=$CustomSnkLocation /p:SignAssembly=true
```

The `-f netstandard2.0` switch which we used for parallel target building forces that it finds every project with that target in the solution. Since we have a multitargeted NetStandard.csproj and a single sln for Netstandard that doesn't have netstandard2.0 broken out it is required here.

Reply:
Ok I'm going to revert this for now to not further derail things but I have questions 😂

Reply:
Done